xml/System.Threading/SemaphoreSlim.xml

```diff
@@ -822,11 +822,7 @@ If a thread or task is blocked when calling <xref:System.Threading.SemaphoreSlim
  ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentOutOfRangeException">
-          <paramref name="timeout" /> is a negative number other than -1, which represents an infinite timeout.
-
--or-
-
-<paramref name="timeout" /> is greater than <see cref="F:System.Int32.MaxValue">Int32.MaxValue</see>.</exception>
+          <paramref name="timeout" /> is a negative number other than -1, which represents an infinite timeout.</exception>
```

Comment:
```suggestion
          <paramref name="timeout" /> is a negative number other than -1, which represents an infinite timeout.

-or-

In .NET 9 and earlier versions, <paramref name="timeout" /> is greater than <see cref="F:System.Int32.MaxValue">Int32.MaxValue</see>.</exception>
```
We have one set of docs for all .NET versions. Behavior differences between versions needs to called out so that the docs are applicable to all .NET version. Here is an example from existing System.Threading docs for reference: https://github.com/dotnet/dotnet-api-docs/blob/b314392d22b44ed9a5143a6984bc15636b096ea0/xml/System.Threading/Interlocked.xml#L1353